plugins/slo/pkg/slo/slo_test.go

```diff
@@ -73,7 +75,6 @@ var _ = Describe("Converting SLO information to Cortex rules", Ordered, Label(te
 			},
```

Comment:
This test suite should be labeled `integration`, also need to use the string literal labels instead of the constants. `Label("integration", "slow")`